core/nullDC.cpp

```diff
@@ -134,7 +134,7 @@ void LoadSpecialSettings()
 	safemode_game = false;
 	tr_poly_depth_mask_game = false;
 	extra_depth_game = false;
-	
+
```

Comment:
Please try to avoid white-space changes in future PRs, short diffs are easier to read. For formatting and white-space, it's best to open a PR with just those changes.